assets/sass/05-blocks/latest-posts/_style.scss

```
@@ -16,6 +16,7 @@
 	}
 
 	&.is-grid {
+		word-wrap: anywhere;
```

This doesn't work in IE or Safari, but this should work
```suggestion
		word-wrap: break-word;
		word-break: break-word;
```

---

Good catch, thank you!